src/ScrumMaster/Jira/Tickets.php

```diff
@@ -32,6 +33,17 @@ public static function fromArrayIssues(array $issues): array
         return $jiraTickets;
     }
 
+    public function __construct(string $fieldStoryPoints)
+    {
+        Assert::notEmpty($fieldStoryPoints);
+        self::$fieldStoryPoints = $fieldStoryPoints;
```

Comment:
You are setting a static value via a constructor? It makes no sense.

What about removing the static context about this class and use it as an individual instance? Think about passing it from the top to the bottom. Don't fear refactoring!